### PR TITLE
Added bugzilla module (fixes #27)

### LINKED ISSUE #27: Create a bugzilla module with a function to get the crash signatures from a bugzilla bug
We will need to use the bugzilla bugs to evaluate the results of the tool. Developers manually clustered signatures together, we need to check that our tools clusters the same signatures together.

As a first step, we need a function that gets crash signatures from a bugzilla bug.

Some documentation is at http://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html.
Example query: https://bugzilla.mozilla.org/rest/bug/1308863.

### PR DESCRIPTION
**Pull request by ray-pankaj**

### CODE REVIEW

**Comment by ray-pankaj** [comment]
This works in ubuntu but not OS X

File: bugzilla.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import utils
+
+
+def get_signatures_from_bug(bug_id):
+    url = 'https://bugzilla.mozilla.org/rest/bug'
+    response = utils.get_with_retries(url, params={'id': bug_id})
+    response.raise_for_status()
+    return response.json()['bugs'][0]['cf_crash_signature']
```
Comment (marco-c) [inline bugzilla.py]: cf_crash_signature can contain multiple signatures, we have to split the field in an array. See for example https://bugzilla.mozilla.org/show_bug.cgi?id=1333486.

**Comment by marco-c** [comment]
> This works in ubuntu but not OS X

Strange, the code is quite simple. What happens on OS X?

**Comment by ray-pankaj** [comment]
On os x it returns an ssl error
requests.exceptions.SSLError: [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:590)

**Comment by marco-c** [comment]
> On os x it returns an ssl error

Can you try reinstalling the dependencies (pip install -r requirements.txt)? I think c960bc4039544cd32f6e206bc02d5ace1ae59d84 might have fixed this problem.

File: bugzilla.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import utils
+
+
+def get_signatures_from_bug(bug_id):
+    url = 'https://bugzilla.mozilla.org/rest/bug'
+    response = utils.get_with_retries(url, params={'id': bug_id})
+    response.raise_for_status()
+    return [str(sig)[1: len(sig) - 1] for sig in response.json()['bugs'][0]['cf_crash_signature'].split('\r\n')]
```
Comment (marco-c) [inline bugzilla.py]: We should remove the `[@ ` and `]` characters from the signature. There are several options, as the format is not always the same:
1. [@ SIGNATURE ]
2. [@SIGNATURE]
3. [@ SIGNATURE]
4. [@SIGNATURE ]

So, for each string, we should remove `[@` and `]` from the beginning and end of the string and then trim the result.

**Comment by ray-pankaj** [comment]
Installing requests[security] worked. Thanks!

**Comment by ibrahimsharaf** [comment]
Wouldn't it be better if this function is added in crash_similarity.py?

**Comment by ray-pankaj** [comment]
We'll need more functions in the module to evaluate results of the tool, so its probably best to not clutter crash_similarity.py and keep them separate.

File: bugzilla.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import utils
+
+
+def get_signatures_from_bug(bug_id):
+    url = 'https://bugzilla.mozilla.org/rest/bug'
+    response = utils.get_with_retries(url, params={'id': bug_id})
+    response.raise_for_status()
+    return [str(sig)[2: len(sig) - 1].strip() for sig in response.json()['bugs'][0]['cf_crash_signature'].split('\r\n')]
```
Comment (marco-c) [inline bugzilla.py]: Can you do `sig.strip(' [@]')` instead of `str(sig)[2: len(sig) - 1].strip()`, just to be extra safe.

**Pull request merged**